src/tools/tidy/src/style.rs

```diff
@@ -519,8 +519,11 @@ pub fn check(path: &Path, bad: &mut bool) {
                         .any(|directive| matches!(directive, Directive::Ignore(_)));
                 let has_alphabetical_directive = line.contains("tidy-alphabetical-start")
                     || line.contains("tidy-alphabetical-end");
-                let has_recognized_directive =
-                    has_recognized_ignore_directive || has_alphabetical_directive;
+                let has_other_tidy_ignore_directive =
+                    line.contains("ignore-tidy-target-specific-tests");
```

Comment:
Question: this tidy ignore directive is new right? Can you please also describe this in `rustc-dev-guide` (in-tree) so other contributors might have a chance to find out about it?

Reply:
Good idea: added a note and created a section for tidy-related test directives.